### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by Sitanshu-AI**
Added Code Of Conduct ,it will be helpful to ones who are new to Open Source.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+https://www.contributor-covenant.org/version/2/0/code_of_conduct.html.
+
+Community Impact Guidelines were inspired by [Mozilla's code of conduct
+enforcement ladder](https://github.com/mozilla/diversity).
+
+[homepage]: https://www.contributor-covenant.org
+
+For answers to common questions about this code of conduct, see the FAQ at
+https://www.contributor-covenant.org/faq. Translations are available at
+https://www.contributor-covenant.org/translations.
```
Comment (dhwaj1902) [inline CODE_OF_CONDUCT.md]: Please remove these lines.

Comment (Sitanshu-AI) [reply CODE_OF_CONDUCT.md]: ok

Comment (Sitanshu-AI) [reply CODE_OF_CONDUCT.md]: Done please Check

**Comment by dhwaj1902** [comment]
Please fill the form available on slack channel

**Pull request merged**